includes/Core/Authentication/Authentication.php

```diff
@@ -372,11 +345,11 @@ function( $user ) {
 					$user['user']['full_name'] = isset( $profile_data['full_name'] ) ? $profile_data['full_name'] : null;
 				}
 
-				$user['connectURL']        = esc_url_raw( $this->get_connect_url() );
-				$user['hasMultipleAdmins'] = $this->has_multiple_admins->get();
-				$user['initialVersion']    = $this->initial_version->get();
-				$user['userInputState']    = $this->user_input_state->get();
-				$user['verified']          = $this->verification->has();
+				$user['connectURL']          = esc_url_raw( $this->get_connect_url() );
+				$user['hasMultipleAdmins']   = $this->has_multiple_admins->get();
+				$user['initialVersion']      = $this->initial_version->get();
+				$user['isUserInputComplete'] = ! $this->user_input->are_settings_empty();
```

Comment:
Somewhat of a nitpick, but I feel like using the past tense "completed" would be more consistent with other state and selectors we have. Similar to below (`verified`), or `isAuthenticated`. What do you think? I realize that change might touch a lot of this PR but should be fairly quick and easy to do.